tools/export/makefile/__init__.py

```diff
@@ -200,7 +200,7 @@ class GccArm(Makefile):
 
     @staticmethod
     def prepare_lib(libname):
-        return "-l:" + libname
+        return "-l" + libname[3:-2]
```

Comment:
Isn't the name generated by prefix + name + suffix because its a gcc naming convention?